infrastructure/modules/aks-resources/variables.tf

```
@@ -0,0 +1,30 @@
+variable "aks_node_resource_group" {
+  type        = string
+  description = "AKS node resource group name"
+}
+
+variable "azurerm_kubernetes_cluster_id" {
+  type        = string
+  description = "AKS cluster resource id"
+}
+
+variable "flux_release_tag" {
+  type        = string
+  default     = "latest"
+  description = "Flux release tag on oci images"
```

Personal opinion, but makes it more clear for me what this is
```suggestion
  description = "OCI image that Flux should watch and reconcile"
```